src/checkpoint/container/view_serialize.h

```diff
@@ -475,7 +521,7 @@ inline void serialize_const(SerializerT& s, Kokkos::View<T,Args...>& view) {
   using T_non_const = typename ViewType::traits::non_const_data_type;
   Kokkos::View<T_non_const,Args...> tmp_non_const(view.label(), view.layout());
   if (s.isPacking() || s.isSizing()) {
```

Comment:
@lifflander As an aside to this PR, why does this call the copy for the sizing case?

Reply:
I looked at the history, and this has been there from the earliest sketch. It still doesn't make sense to me.

Reply:
I think that the isSizing can be removed.

Reply:
Will remove `|| isSizing()`